docs/setup/main_setup.md

```diff
@@ -29,16 +29,17 @@ Use your distribution's package manager to install the following dependencies:
     1. Install dependencies
 
         ```sh
-        sudo apt install git git-lfs build-essential libssl-dev pkg-config libclang-dev rsync cmake libhdf5-dev python3
+        sudo apt install git git-lfs build-essential libssl-dev pkg-config libclang-dev rsync cmake libhdf5-dev python3 libasound2-dev libluajit-5.1-dev
         ```
 
     1. Install Webots
+        Download webots from [https://cyberbotics.com/](https://cyberbotics.com/) the XXXX.deb file and install it with
 
         ```sh
-        sudo snap install webots
+        sudo dpkg -i XXXX.deb
         ```
 
-    1. Install rust toolchain
+    2. Install rust toolchain
```

Comment:
This enumeration is now 1. 1. 2. 
I'm not sure what that does, but if you just use 1. all the time, it will be enumerated correctly.

Reply:
Done